manager/controlapi/service.go

```diff
@@ -257,6 +257,10 @@ func (s *Server) UpdateService(ctx context.Context, request *api.UpdateServiceRe
 		}
 		service.Meta.Version = *request.ServiceVersion
 		service.Spec = *request.Spec.Copy()
+
+		// Reset update status
+		service.UpdateStatus = nil
```

Comment:
I think it's important to do it in controlapi. The orchestrator has no way of knowing if the user updated the service, or it was updated for some internal reason (for example, the updater setting the update state to `UPDATING`).